site/en/r2/tutorials/load_data/text.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Maybe show how to use SubwordsTokenizer too, since that's easier to build and it used more than a normal tokenizer.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>